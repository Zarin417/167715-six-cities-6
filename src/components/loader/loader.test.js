import React from "react";
import {render} from "@testing-library/react";
import Loader from "./loader";

it(`Loader should render correctly`, () => {
  const {container} = render(
      <Loader />
  );

  expect(container).toMatchSnapshot();
});
