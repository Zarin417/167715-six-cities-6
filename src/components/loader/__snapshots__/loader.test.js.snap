// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loader should render correctly 1`] = `
<div>
  <div
    class="loader"
  >
    <div
      class="loader__dual-ring"
    />
    <p>
      Loading...
    </p>
  </div>
</div>
`;
